x-pack/legacy/plugins/ml/common/constants/license.ts

```diff
@@ -4,8 +4,7 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-
-export const LICENSE_TYPE = {
-  BASIC: 0,
-  FULL: 1,
-};
+export enum LICENSE_TYPE {
+  BASIC,
+  PLATINUM,
```

Comment:
Maybe it is better keeping this as `FULL` as a trial license will also enable the full functionality as well as platinum.